src/components/HeaderDropdown.tsx

```diff
@@ -31,8 +36,14 @@ export default function HeaderDropdown() {
     },
   ]
 
+  const navigateTo = (path: string) => {
+    requestAnimationFrame(() => {
+      setLocation(path)
+    })
+  }
+
   return (
-    <DropdownMenu>
+    <DropdownMenu open={open} onOpenChange={setOpen}>
```

Comment:
```suggestion
    <DropdownMenu>
```

I think you can get rid of these, cause the radix is already handling the state of opening and closing internally?

Reply:
Yea I already tried it, but it's working along with PrefetchPageLink